scripts/consensus.py

```diff
@@ -33,21 +45,36 @@ def concatenate_data(profile_list: list) -> pd.DataFrame:
         concat_df.columns.str.startswith("Metadata")
     ]
     concat_metadata_df = concat_df.loc[:, concat_metadata_cols]
+
+    # concatenating metadata
     concat_df = concat_df.drop(concat_metadata_cols, axis="columns")
-    concat_df = pd.concat([concat_metadata_df, concat_df])
+    concat_df = pd.concat([concat_metadata_df, concat_df], axis="columns")
 
     # dropping columns with na values
     na_cols = get_na_columns(concat_df, cutoff=0)
     concat_df = concat_df.drop(na_cols, axis="columns")
 
-    return concat_df
+    # generating consensus profile
+    x_median_df = consensus(
```

Comment:
```suggestion
    x_consensus_df = consensus(
```

No requirement that it is median